src/storages/PerpStorage.sol

```diff
@@ -328,6 +330,10 @@ contract PerpStorage is OwnableUpgradeable, ReentrancyGuardUpgradeable, IPerpSto
     return movingWindowInterval > 0 ? (block.timestamp / movingWindowInterval) * movingWindowInterval : block.timestamp;
   }
 
+  function setLastIncreaseSize(bytes32 positionId, uint256 tradeSize) external onlyWhitelistedExecutor {
+    lastIncreaseSizeByPositionId[positionId] = tradeSize;
```

Comment:
Log here too

Reply:
Remove this function as the attribute is inside struct